app/src/main/java/com/spartahack/spartahack17/Retrofit/ISpartahackAPIService.java

```diff
@@ -0,0 +1,22 @@
+package com.spartahack.spartahack17.Retrofit;
+
+import com.spartahack.spartahack17.Model.Session;
+
+import retrofit2.http.Body;
+import retrofit2.http.DELETE;
+import retrofit2.http.POST;
+import retrofit2.http.Path;
+import rx.Observable;
+
+/**
+ * Created by ryancasler on 9/19/16.
+ * SpartaHack2016-Android
+ */
+public interface ISpartahackAPIService {
```

Comment:
also looks like the repo name is still SpartaHack2016-Android in your IDE settings haha